google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java

```diff
@@ -134,8 +135,14 @@ public File get() {
         new GenerateConfigActionListener(project, "Dockerfile", new Supplier<File>() {
           @Override
           public File get() {
+            File deploymentFile;
+            if(deploymentSource.getType() instanceof UserSpecifiedPathDeploymentSourceType) {
+              deploymentFile = new File(userSpecifiedArtifactFileSelector.getText());
```

Comment:
The problem is that we don't know what to return from getFile: this only gets determined when a user manually browses to one. We could manually set it, e.g. add state to the class. But I'm not sure if thats better.